### ISSUE TITLE
Not really an issue... Just a suggestion...

### ISSUE THREAD

**Issue by evanevery** [schlae/graphics-gremlin#11]
I'm trying get some old floptical drives running on several ISA-based platforms I have been working with.  I came across this project and I'm glad to see someone is working to make NEW hardware for OLD machines.  Anyway...

How about an ISA based video card that goes straight to HDMI?

While this might seem relatively trivial, anyone who is trying to hook up an old ISA/VGA card via a simple VGA->HDMI converter knows - its not.  The unfortunate fact of the matter is that most new HDMI monitors don't support the necessary 700x420@70Hz signal used by BIOS/DOS.  To make matters even worse, most current HDMI upscalers won't even do this anymore.  (I.E. The $200 Startech VGA2HDPRO2 which replaced the previous model VGA2HDMIPRO (which did) doesn't...)

Not expecting you to do this but seems like you might be interested....

Now, if only we had an ISA based controller which supported direct attachment (on board) of SDCards for both floppy and Hard Drive emulation...  Maybe add a serial, parallel, game port, and sound function is suitably motivated.  Maybe put all of this on the same ISA card with upscaled HDMI output and you have a single ISA card ready to provide any/all retro PC services as needed.

Sorry if this is not appropriate for posting here....

**Comment by darkstar** [comment]
Indeed, at least an integrated RGB2HDMI with a connector for the RPi would be a welcome feature. However, the VGA port is probably future-proof enough for at least a couple more years, so it's not that much of an issue.

Until then, you can pick up the RGB signals and feed them to an external RGB2HDMI yourself if you need to...

**Comment by evanevery** [comment]
The VGA port is not the issue.  The problem is that most new HDMI monitors (even those WITH VGA ports) simply don't support 700x420 (the DOS/BIOS) standard anymore.  If we went directly to HDMI then we could have better control of our output resolutions (and not require additional VGA to HDMI converters which have ALSO generally dropped support for 700x420)

IOW:  If we start with VGA, then we are at the mercy of the VGA to HDMI converters/scalers (either external or built-in to monitors).  If we start with HDMI, and scale as needed, then we are no longer dependent on hardware which is effectively no longer available.

**Comment by yeokm1** [comment]
@evanevery I think you will be happy to know I have made that HDMI improvement.

https://github.com/yeokm1/graphics-gremlin-hdmi
